### ISSUE TITLE
Refactor timeCommitmentMin/Max

### ISSUE THREAD

**Issue by aleesaan** [xrnl/aginun#225]
Instead of having two properties in the role model, we should refactor them into an object with [this format](https://github.com/xrnl/aginun/blob/9d1a1e75d8dea2e6854635dcb0f966828aa0b59f/src/constants/timeCommitments.ts#L1).

**Comment by SamLubbers** [comment]
@aleesaan I agree that the frontend and backend should have a consistent data format. Making the backend model have time commitment as a model will make the filtering a bit more difficult, but it's possible. The other option, is to make time commitment in the frontend two separate fields. What do you think is better?

**Comment by aleesaan** [comment]
Yea indeed two separate fields would be even better! Do you have something in mind UI-wise?

**Comment by SamLubbers** [comment]
@aleesaan can you explain a bit more? I was just thinking about how the data is stored, but i think you also have some UI changes in mind

**Comment by aleesaan** [comment]
> The other option, is to make time commitment in the frontend two separate fields. What do you think is better?
  
I was thinking along the lines of this one! We could use the range slider we have for the filters, and store the time commitments in an array like `[min, max]`. Then we can simply access them by index when creating the role, and we won't need any BE changes.
